client/app/tasks/[id]/edit/page.tsx

```diff
@@ -1,5 +1,34 @@
 'use client'
 
-export default function EditTask() {
-  return <p>This is edit page</p>
+import { FormContainer, Loading } from '@/_components/layout'
+import { Heading } from '@/_components/ui'
+import { useGetHouseQuery } from '@/_hooks/api'
+import { EditTaskForm } from '@/tasks/components'
+import { useGetTaskQuery } from '@/tasks/hooks/api'
+
+export default function EditTask({ params }: { params: { id: string } }) {
+  const taskId = Number(params.id)
+  const { isLoading: isLoadingHouse } = useGetHouseQuery()
+  const {
+    data: task,
+    isError: isTaskError,
+    isLoading: isLoadingTask,
+  } = useGetTaskQuery(taskId)
+
+  // If the user tries to access to the url that has invalid taskId, show an error component
```

Comment:
I think you don't need this comment?

Reply:
Deleted it!
https://github.com/HAK2024/fairy-share/pull/84/commits/184a595460639b4f3543b67755cd993921f961c6